lib/google_places/spot.rb

```diff
@@ -1,5 +1,5 @@
 require 'google_places/review'
-
+require 'google_places'
```

Comment:
We can chalk this up to being a Ruby noobie :P .  Pulling this out for next commit